### DISCUSSION TITLE
请问本地部署失败 我该如何操作

### DISCUSSION THREAD

**Discussion by Cyiming** [JimmyLv/BibiGPT#50] [Q&A]
我是把代码打包到本地后 运行npm run dev

没成功提示
![image](https://user-images.githubusercontent.com/45073586/225196916-e44cd678-7f5f-41c7-90e6-9851a3b51c44.png)

**Comment by conglinyizhi** [comment]
看起来是装载 `next.config.js` 的时候出现了错误，但是我不能确定到底是因为什么出现的这个问题，可以尝试根据错误信息后提供的链接去解决
